src/js/index.js

```diff
@@ -1,5 +1,7 @@
 // Component imports
-import { defaultComp } from './components/default';
+import { defaultComp } from './components/default.js';
+import { initialMessageComp } from './components/initialmessage';
+
```

Comment:
Please use camalcase to name the JS files 

'initialMessage'

Reply:
Done!!!  camelCase implemented.